src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -486,8 +486,12 @@ protected function denormalizeParameter(\ReflectionClass $class, \ReflectionPara
      *
      * @internal
      */
-    protected function createChildContext(array $parentContext, $attribute/*, string $format = null */)
+    protected function createChildContext(array $parentContext, $attribute/*, ?string $format */)
     {
+        if (\func_num_args() < 3) {
+            @trigger_error(sprintf('Method %s::%s() will have a 3rd `?string $format` argument in version 5.0. Not defining it is deprecated since Symfony 4.3.', \get_class($this), __FUNCTION__), E_USER_DEPRECATED);
```

Comment:
`Method "%s::%s()" will have a 3rd "?string $format" argument in version 5.0; not defining it is deprecated since Symfony 4.3.`

Reply:
fixed

Reply:
It still looks like very different from my proposal :)

Reply:
sorry, should have copied the whole line instead of editing. i hope i now catched everything.